assets/css/style.css

```diff
@@ -0,0 +1,281 @@
+/** Import Font **/
+
+@import url("https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i");
+
+/** Import Font  End **/
+
+/** Shared Button Effects **/
+.welcome-section {
+  padding-top: 50px;
+  height: 100%;
+  display: block;
+}
+
+.btn-see-project {
+  border: 1px solid #396df2;
+  padding: 7px;
+  border-radius: 8px;
+  font-size: 16px;
+  font-weight: 300;
+  text-transform: capitalize;
+  font-family: 'Poppins', sans-serif;
+  transition-duration: 0.4s;
+}
+
+a.btn-see-project {
```

Comment:
- [ ] please kindly follow the Figma styling for the `see live and see source buttons` as the previous reviewer noted especially for the mobile version. increase the size and make it centered. 
your version 👇🏾 
![Screenshot (142)](https://user-images.githubusercontent.com/65298048/174022248-62436063-90c3-4f3c-9c4e-3ee4af24f3dd.png)
figma version 👇🏾 

![Screenshot (141)](https://user-images.githubusercontent.com/65298048/174022329-4d42964f-a47c-43b6-b352-384e183d3daa.png)